unittests/integration/get_state_tester.cpp

```diff
@@ -7,6 +7,7 @@
  *******************************************************************************/
 
 #include "CUDAQTestUtils.h"
+#include <Eigen/Dense>
```

Comment:
If anyone ever wanted to get the data from the state they would have to ensure they have Eigen installed.